src/sentry/api/endpoints/organization_events_trends_v2.py

```diff
@@ -351,6 +363,77 @@ def get_stats_data_for_trending_events(results):
                 else {},
             }
 
+        def fingerprint_regression(trending_event):
+            return hashlib.sha1(
+                (
+                    f'p95_transaction_duration_regression-{trending_event["transaction"]}-experiment'
+                ).encode()
+            ).hexdigest()
+
+        def send_occurrence_to_plaform(found_trending_events):
+            nonlocal top_trending_transactions
+            qualifying_trends = []
+            for trend in found_trending_events:
+                if (
+                    request.GET.get("statsPeriod", None) == "14d"
+                    and trend.get("change", None) == "regression"
+                    # trends >50%
+                    and (trend.get("trend_percentage", None) - 1) >= 0.5
```

Comment:
This should be a tunable parameter

Reply:
I'm going to follow up with that 👍 I'm trying to unblock Nar as soon as possible. if this causes any problems we can turn off the feature flag